src/pages/studies/SidebarFilters.tsx

```diff
@@ -32,6 +35,26 @@ const keyEnhanceBooleanOnly = (key: string) => {
   }
 };
 
+const onSearch = (search: string) => {
+  const searchFields = ['search'];
+  searchFields.forEach((f) =>
```

Comment:
why using a `foreach` since there's only one element?

Reply:
good call . yeah there were a couple b4 and they were merged into a single new field (search). Ill refactor.